corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -421,6 +421,7 @@ hqDefine("users/js/mobile_workers",[
             location_url: initialPageData.reverse('location_search'),
             require_location_id: !initialPageData.get('can_access_all_locations'),
             strong_mobile_passwords: initialPageData.get('strong_mobile_passwords'),
+            two_stage_provisioning_enabled: initialPageData.get('two_stage_provisioning_enabled'),
```

Comment:
All feature flags are available in js, here's an example: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static/app_manager/js/details/screen_config.js#L1170

Reply:
https://github.com/dimagi/commcare-hq/pull/26946/commits/57ca691348420c170649f633bd3b5387a6d5a842